.github/workflows/build.yml

```diff
@@ -4,7 +4,7 @@ jobs:
   build:
     strategy:
       matrix:
-        java: [8-jdk, 11-jdk, 16-jdk, 17-jdk, 19-jdk]
+        java: [8-jdk, 11-jdk, 16-jdk, 17-jdk, 20-jdk]
```

Comment:
Do we keep 16 as minecraft eagerly bumped for records before? I think everything 16 offers is already covered by 17.

Reply:
Minecraft has versions that run on JDK 16, I think for now we keep it. In the future I do plan to bump loaders min build version to 17 (While still targgeting 8) so we can remove it then.